tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Add unstable repo
+  apt_repository:
+    repo: deb http://deb.debian.org/debian/ unstable main
+
+- name: Pin unstable repo
+  copy:
+    content: |
+      Package: *
+      Pin: release a=unstable
+      Pin-Priority: -1
+    dest: /etc/apt/preferences.d/limit-unstable
+
+- name: Install WireGuard
+  apt:
+    update_cache: True
```

Comment:
I think the `apt_repository` module updates the apt cache on changes, so this may not be necessary.

Reply:
Apparently [it does](https://docs.ansible.com/ansible/latest/modules/apt_repository_module.html). I will remove line 16.